### PR TITLE
[Feature-8385][Server] Custom timezone - unified config timezone by spring config

### PR DESCRIPTION
**Pull request by jim-parsons**
## Purpose of the pull request

this pr unifies timezone by spring config

## Brief change log

1. add spring jackson timezone config
2. add new json utils with spring context and unit test

complete #8385

## Verify this pull request

This change added tests and can be verified as follows:

- add `JSONUtilsTest` to verify this pr

### CODE REVIEW

**Comment by jim-parsons** [comment]
because the function of jsonutils uesed almost everywhere, so now i just write some basic code to make sure the direction is not wrong ~
plz give some advice @caishunfeng , thx~

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JacksonUtils.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.CollectionType;
+
+@Component
+public class JacksonUtils {
```
Comment (caishunfeng) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JacksonUtils.java]: Why not modify the `JSONUtils` directly? I think it wll need to modify a lot of places if use a new json util.

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/config/JacksonObjectMapperConfig.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+     * use spring config to set timeZone
+     *
+     * @param objectMapper objectMapper
+     */
+    public JacksonObjectMapperConfig(ObjectMapper objectMapper) {
+        objectMapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false)
+            .configure(ACCEPT_EMPTY_ARRAY_AS_NULL_OBJECT, true)
+            .configure(READ_UNKNOWN_ENUM_VALUES_AS_NULL, true)
+            .configure(REQUIRE_SETTERS_FOR_GETTERS, true)
+            .setDateFormat(new SimpleDateFormat(Constants.YYYY_MM_DD_HH_MM_SS));
```
Comment (caishunfeng) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/config/JacksonObjectMapperConfig.java]: Will it use the spring config timezone by default?

**Comment by caishunfeng** [comment]
> because the function of jsonutils uesed almost everywhere, so now i just write some basic code to make sure the direction is not wrong ~ plz give some advice @caishunfeng , thx~

It's OK, I had submited some review comment. BTW, the annotation @JsonFormat should be handled too.

**Pull request closed without merging**